app/utils/userAgentInfo.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import UAParser from 'ua-parser-js';
```

Comment:
Probably the user agent is not the best way to do this. After some googling, it seems like `getURL('/')` (https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/getURL) to get whether or not your extension prefix is `moz-extension` or `chrome-extension`.
The added benefit of this method is it will work for any Chrome-like browser whereas your solution would fail to detect Chromium which doesn't use "Chrome" as the browser name.

The downside is that this solution won't work if Yoroi is ever hosted as a website (which we don't plan to do anyway) but having the browser name in the logs would be helpful so maybe we should:

1) Use the method I mentioned to find which Trezor manifest to load and in the case that it's neither Firefox nor Chrome, just use Chrome as the manifest (?)
2) Use this library to add browser / OS name to the logs

Reply:
@SebastienGllmt 
Yeah I agree.

Few things i would like to change here:
I think `browser.runtime.getURL()` is only Mozilla/Firefox global variable so in case of chrome it will be `undefined`

Instead use `window.location.protocol` which will give result like: 
1. Chrome like: `chrome-extension:`
2. Firefox: `moz-extension:`